core/src/storage_stage.rs

```diff
@@ -126,6 +126,7 @@ impl StorageState {
         if index < replicator_map.len() {
             replicator_map[index]
                 .iter()
+                .map(|(id, _)| id)
```

Comment:
`.keys()` here?

Reply:
oh I can't `.keys()` since each index in the vec is a map and vec.iter().keys() doesn't know to run on each index. That's why the map.

Reply:
ah ok